frontend/public/components/pdb/pdb.tsx

```diff
@@ -0,0 +1,272 @@
+      {
+        title: t('public~Name'),
+        sort: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnInfo[0].className },
+        id: tableColumnInfo[0].id,
+      },
+      {
+        title: t('public~Namespace'),
+        sort: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnInfo[1].className },
+        id: tableColumnInfo[1].id,
+      },
+      {
+        title: t('public~Selector'),
+        sort: 'spec.selector',
+        transforms: [sortable],
+        props: { className: tableColumnInfo[2].className },
+        id: tableColumnInfo[2].id,
+      },
+      {
+        title: t('public~Availability'),
+        sort: 'spec.minAvailable',
+        transforms: [sortable],
+        props: { className: tableColumnInfo[3].className },
+        id: tableColumnInfo[3].id,
+      },
+      {
+        title: t('public~Allowed disruptions'),
+        sort: 'status.disruptionsAllowed',
+        transforms: [sortable],
+        props: { className: tableColumnInfo[4].className },
+        id: tableColumnInfo[4].id,
+      },
+      {
+        title: t('public~Created'),
+        sort: 'metadata.creationTimestamp',
+        transforms: [sortable],
+        props: { className: tableColumnInfo[5].className },
+        id: tableColumnInfo[5].id,
+      },
+      {
+        title: '',
+        props: { className: tableColumnInfo[6].className },
+        id: tableColumnInfo[6].id,
+      },
+    ],
+    [t],
+  );
+
+  const [columns] = useActiveColumns({ columns: pdbTableColumns });
+
+  return (
+    <VirtualizedTable<PodDisruptionBudgetKind>
+      {...props}
+      aria-label={t('public~PodDisruptionBudgets')}
+      columns={columns}
+      Row={PodDisruptionBudgetTableRow}
+    />
+  );
+};
+export const PodDisruptionBudgetsPage: React.FC<PodDisruptionBudgetsPageProps> = ({
+  kind,
+  namespace,
+  showTitle = true,
+}) => {
+  const { t } = useTranslation();
+  const [resources, loaded, loadError] = useK8sWatchResource<PodDisruptionBudgetKind[]>({
+    kind,
+    isList: true,
+    namespaced: true,
+    namespace,
+  });
+
+  const [data, filteredData, onFilterChange] = useListPageFilter(resources);
+  const createProps = `/k8s/ns/${namespace || 'default'}/poddisruptionbudget/form`;
+
+  return (
+    <>
+      <ListPageHeader title={showTitle ? t(PodDisruptionBudgetModel.labelPluralKey) : undefined}>
+        <ListPageCreateLink to={createProps}>
+          {t('public~Create PodDiscruptionBudget')}
+        </ListPageCreateLink>
+      </ListPageHeader>
+      <ListPageBody>
+        <ListPageFilter data={data} loaded={loaded} onFilterChange={onFilterChange} />
+        <PodDisruptionBudgetList
+          data={filteredData}
+          unfilteredData={resources}
+          loaded={loaded}
+          loadError={loadError}
+        />
+      </ListPageBody>
+    </>
+  );
+};
+
+const PodDisruptionBudgetDetails: React.SFC<PodDisruptionBudgetDetailsProps> = ({
```

Comment:
```suggestion
const PodDisruptionBudgetDetails: React.FC<PodDisruptionBudgetDetailsProps> = ({
```

SFC is deprecated. Not sure what IDE you're using, but hopefully you're seeing strikethrough names sorta like this:
<img width="684" alt="image" src="https://user-images.githubusercontent.com/8126518/159767639-0dca5a87-094a-4ff8-90cc-e1f0317d3e09.png">